modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/schema/SqlSchemaManager.java

```diff
@@ -27,8 +28,11 @@
 public interface SqlSchemaManager {
     /**
      * Returns a required schema if specified, or default schema otherwise.
+     *
+     * @param schema Schema name.
+     * @param ts Transaction timestamp to obtain the effective schema version.
      */
-    SchemaPlus schema(@Nullable String schema);
+    SchemaPlus schema(@Nullable String schema, HybridTimestamp ts);
```

Comment:
AFAIK, we agree to use long timestamp HybridTimeStamp.longValue()

Reply:
https://github.com/apache/ignite-3/pull/2100/files#diff-93ec945a50393605375cb8053d980c3c916aa31bfe6e2479c7c02b459aa2f3e7